src/components/Sections/SectionChart/SectionChart.js

```diff
@@ -21,7 +21,7 @@ const filterResults = (chartProperties, rawResults) => {
 };
 
 const SectionChart = ({ type, data: rawData, style, dimensions, legend, chartProperties = {}, legendStyle = {}, sortBy,
-  referenceLineX, referenceLineY, title, stacked, fromDate, toDate, titleStyle }) => {
+  referenceLineX, referenceLineY, title, stacked, fromDate, toDate, titleStyle, showOverflow }) => {
```

Comment:
who is passing the showOverflow prop?

Reply:
reportslayouts to layouts

Reply:
regarding your q of "whats the 300 for anyway?" (cant answer directly there), when we want to see the full chart and full legend, we stretch the height to to the full height of the legend and then add extra fixed height for the chart, as agreed by the UX team